### DISCUSSION TITLE
Implement Playwright test use options

### DISCUSSION THREAD

**Discussion by SmartALB** [artilleryio/artillery#2402] [Artillery]
In Playwright, the use-option can be set to the value for slowMo (reference [here](https://playwright.dev/docs/test-use-options#configuration-scopes)), which prevents clicks from following each other too quickly. My unsuccessful attempts to implement the same for Artillery like this
```js
async function helloFlow(page, _context, _events, test) { 
  test.use({slowMo: 650})
```

 lead me to the conclusion that this is not yet possible.  

Would this be feasible for Artillery in the near future?

**Comment by bernardobridge** [comment]
Hi @SmartALB ! 

Artillery doesn't use Playwright's test runner. We may look into doing that in the future, but currently it is not possible to use features from the `@playwright/test` package's `test` instance. The exception is `test.step()` which is an Artillery-specific API we made that resembles Playwright's own. 

For `slowMo` specifically, as well as several other options that are set with `test.use`, you should be able to pass it under `launchOptions` and it will be set for all scenarios:
- **Example of using `launchOptions` in Artillery config**: https://www.artillery.io/docs/reference/engines/playwright#example-1-turn-off-headless-mode
- **Playwright `launchOptions` reference**: https://playwright.dev/docs/api/class-browsertype#browser-type-launch
